alf/examples/ac_target_navigation.gin

```diff
@@ -37,7 +37,7 @@ num_state_tiles=800
 num_sentence_tiles=800
 
 mlp_layers.activation_fn=@tf.keras.activations.softsign
-mlp_layers.kernel_initializer=@tf.compat.v1.keras.initializers.glorot_uniform()
+mlp_layers.kernel_initializer=@tf.keras.initializers.GlorotUniform
```

Comment:
Missing ()?: GlorotUniform()

Reply:
You are right!  Fixed.  This is scary, the code can still run without complaint..